### PR TITLE
Add box tiny expr to main

### PR DESCRIPTION
**Pull request by aronnoordam**

### CODE REVIEW

File: stem/utils.py [hunk 1]
```diff
@@ -376,6 +376,47 @@ def create_sigmoid_tiny_expr(start_time: float, dt_slope: float, initial_value:
...
+                             end_peak: float,
+                             peak_value: float,
+                             base_value: float,
+                             variable: str = "x") -> str:
+        """
+        Creates a tiny expression for a box function. For more information on tiny expressions,
+        see: https://github.com/codeplea/tinyexpr
+
+        Args:
+            - transition_parameter (float): parameter to control the transition of the box function, |
```
Comment (brunozc) [inline stem/utils.py:391]:
```suggestion
            - transition_parameter (float): parameter to control the transition of the box function, \
```

File: stem/utils.py [hunk 2]
```diff
@@ -376,6 +376,47 @@ def create_sigmoid_tiny_expr(start_time: float, dt_slope: float, initial_value:
...
+            - ValueError: when start peak is larger or equal to end peak
+
+        Returns:
+            - str: tiny expression of the box function
+
+        """
+
+        if start_peak >= end_peak:
+            raise ValueError("Start peak should be smaller than end peak.")
+
```
Comment (brunozc) [inline stem/utils.py:409]: add a ValueError for variable: 
if variable not in ['x', 'y', 'z', 't']:
    raise ValueError(".....")

File: stem/utils.py [hunk 3]
```diff
@@ -376,6 +376,47 @@ def create_sigmoid_tiny_expr(start_time: float, dt_slope: float, initial_value:
...
 
+    @staticmethod
+    def create_box_tiny_expr(transition_parameter: float,
+                             start_peak: float,
+                             end_peak: float,
+                             peak_value: float,
+                             base_value: float,
+                             variable: str = "x") -> str:
+        """
+        Creates a tiny expression for a box function. For more information on tiny expressions,
```
Comment (brunozc) [inline stem/utils.py:387]: This is the hyperbolic approximation of the box function. not the box function

File: stem/utils.py [hunk 4]
```diff
@@ -376,6 +376,47 @@ def create_sigmoid_tiny_expr(start_time: float, dt_slope: float, initial_value:
             return (f"(1 / (1 + e^(-{beta} * (t - {dt_slope} / 2 - {start_time})))) * ({final_value} - {initial_value})"
                     f"+ {initial_value}")
 
+    @staticmethod
+    def create_box_tiny_expr(transition_parameter: float,
```
Comment (brunozc) [inline stem/utils.py:380]: why do we need the box function?

**Review by brunozc — CHANGES_REQUESTED** [review]

File: stem/utils.py (hunk 4 above)
Comment (aronnoordam) [reply stem/utils.py:380]: to make a step function for the moving load, as tables are not there for the moving load

File: stem/utils.py (hunk 1 above)
Comment (aronnoordam) [reply stem/utils.py:391]: | is the linebreak symbol

**Pull request merged**